modules/util/units.js

```diff
@@ -168,3 +171,44 @@ export function decimalCoordinatePair(coord) {
         longitude: wrap(coord[0], -180, 180).toFixed(OSM_PRECISION)
     });
 }
+
+// Return the parsed value  that @mapbox/sexagesimal can't parse
+// return value format : [D, D]  ex:[ 35.1861, 136.83161 ]
+export function dmsMatcher(q) {
+    const matchers = [
+        // D M SS , D M SS  ex: 35 11 10.1 , 136 49 53.8
```

Comment:
These examples would make for great test cases. Would you mind creating a test/spec/util/units.js that tests this and other edge cases? Thanks!